### DISCUSSION TITLE
Support for `patch` file type in markdown

### DISCUSSION THREAD

**Discussion by ayushmanchhabra** [vuejs/vitepress#2177] [Q&A] [answered]
I get this message in the console after building my docs site:

> The language 'patch' is not loaded, falling back to 'txt' for syntax highlighting. (x17)

Is there a way to enable this? If so could someone point me to the right place?

**Comment by microdev1** [comment, marked as answer]
Try `diff` instead, vitepress uses [shiki](https://shiki.matsu.io/) for [syntax highlighting](https://vitepress.dev/guide/markdown#syntax-highlighting-in-code-blocks) and it doesn't list `patch` in the [supported languages](https://github.com/shikijs/shiki/blob/main/docs/languages.md#all-languages).

**Reply by ayushmanchhabra** [reply]
Thanks!
